core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1642,6 +1642,16 @@
 
   public static final String CARBON_SEARCH_MODE_THREAD_DEFAULT = "3";
 
+  /**
+   * compression mode used by lucene for index writing
+   */
+  public static final String CARBON_LUCENE_COMPRESSION_MODE = "carbon.lucene.compression.mode";
```

Comment:
what are the options available for this property?

Reply:
SPEED and COMPRESSION, by default the property value will be SPEED

Reply:
mention it in the description in line 1646